pkg/quota/api/v1/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package v1 is the v1 version of the API.
+// +genconversion=true
```

Comment:
@soltysh clientset generator works differently, it needs to be in types to get the name of the type...